server/clcachesrv.py

```diff
@@ -5,13 +5,16 @@
 import os
 import pickle
 import signal
+import argparse
+import re
 
 import pyuv
 
 class HashCache(object):
-    def __init__(self, loop):
+    def __init__(self, loop, args):
```

Comment:
I think `args` is a bit too generic and way more than `HashCache` needs. How about just passing `excludedDirs` here?